.github/workflows/nonprod.yaml

```diff
@@ -57,3 +67,4 @@ jobs:
           TF_VAR_bot_user_token: ${{ secrets.NONPROD_BOT_TOKEN }}
           TF_VAR_gratibot_recognize_emoji: ":oof:"
           TF_VAR_gratibot_log_level: "debug"
+          GRATIBOT_LIMIT: ${{ inputs.gratibot_limit }}
```

Comment:
Looks like I was mistaken...we don't have a Terraform variable for Gratibot fistbump limits. We would need to add it to the App Service Terraform and create a new variable

https://github.com/liatrio/gratibot/blob/main/tf/appservice.tf#L33-L39

Reply:
I'd be happy to pair on this one